book/page_creation.rst

```diff
@@ -137,7 +137,7 @@ You can even shorten this with the handy :class:`Symfony\\Component\\HttpFoundat
     // --> don't forget this new use statement
     use Symfony\Component\HttpFoundation\JsonResponse;
 
-    class LuckyController
+    class LuckyController extends Controller
```

Comment:
@WouterJ I'd like to ask you something about this `LuckyController`. What do you think about renaming it to `DefaultController`? This code is shown very early in the book and it's linked from the new Symfony Welcome Page, so most of the readers will be newcomers.